### PR TITLE
add faq content

### PR DESCRIPTION
**Pull request by lislis**

### CODE REVIEW

File: app/views/pages/about.html.haml [hunk 1]
```diff
@@ -1,29 +1,88 @@
...
+      %p This is also how the Rubycorns met.
+      %p This is why most study groups focus on attendees who identify as women. But also this varies froom group to group.
+      %p We have no insight or influence in what groups do, how they're run and how they organize themselves.
+      %p You need to find this out for yourself.
+
+
+      %h3 How do I find a group?
+
+      %p
+        Had over to the
```
Comment (sareg0) [inline app/views/pages/about.html.haml]: `had over` -> `head over`

File: app/views/pages/about.html.haml [hunk 2]
```diff
@@ -1,29 +1,88 @@
...
+      %h3 What are study groups?
+
+      %p Study groups are self organized and volunteer run groups of people who want to learn to code.
+      %p They usually meet weekly but that can vary from group to group.
+      %p
+        The concept of study groups can to be after a
+        =link_to('RailsGirls Berlin', '//railsgirlsberlin.de')
+        workshop. The attendees wanted to keep learning together.
+      %p This is also how the Rubycorns met.
+      %p This is why most study groups focus on attendees who identify as women. But also this varies froom group to group.
```
Comment (sareg0) [inline app/views/pages/about.html.haml]: little typo here: `froom` -> `from`

File: app/views/pages/about.html.haml [hunk 3]
```diff
@@ -1,29 +1,88 @@
...
+      %p
+        Had over to the
+        = link_to('groups page', groups_path)
+        and use the filter to search for study groups meeting near you.
+      %p You can also start your own!
+
+      %h3 How can I join a group?
+
+      %p On a group's detail page you can find information on where and when a group meets.
+      %p It's nice to drop the group a line to announce your coming so that they can prepare for new people.
```
Comment (sareg0) [inline app/views/pages/about.html.haml]: `to announce your coming` -> `to announce you're coming`

**Comment by lislis** [comment]
@sareg0 thanks!

**Comment by lislis** [comment]
@sareg0 can u review again plz??

**Comment by sareg0** [comment]
YAS!

**Pull request merged**